src/components/utils/jwt.js

```diff
@@ -1,25 +1,23 @@
 import { currentSeconds } from './index'
 
-const parseJwt = (token) => {
-  const base64Url = token.split('.')[1]
-  const base64 = base64Url.replace('-', '+').replace('_', '/')
-  return JSON.parse(atob(base64))
+export const parseJwt = (token) => {
+  let parsedJwt = null
+  try {
+    const base64Url = token.split('.')[1]
+    const base64 = base64Url.replace('-', '+').replace('_', '/')
+    parsedJwt = JSON.parse(atob(base64))
+  } catch {}
+  return parsedJwt
 }
 
+export const valideJwT = (token) => token && parseJwt(token)
```

Comment:
Typo here. Should be `validJWT`. Though I personally would prefer `isValidJWT` as a name (minor detail).

Reply:
Good spot, thanks